src/Microsoft.ML.CodeGen/CodeGenerator/CSharp/CodeGenerator.cs

```diff
@@ -282,6 +310,7 @@ private static string GeneratPredictProjectFileContent(string namespaceValue, bo
 
         private string GeneratePredictProgramCSFileContent(string namespaceValue)
         {
+            var features = _columnInferenceResult.TextLoaderOptions.Columns;
```

Comment:
This is confusing, two features variables that vary just by case of the first letter.

Reply:
resolved
features => featuresList